doc/run.md

```diff
@@ -15,8 +15,11 @@ This document instructs you on how to run Nanvix.
 
 ```bash
 # Run Nanvix in QEMU with default parameters:
-# make TARGET=x86 LOG_LEVEL=warn MACHINE=qemu-pc TIMEOUT=90 run
+# make TARGET=x86 MACHINE=qemu-pc VERBOSE=no RELEASE=no TIMEOUT=10 FEATURES= run
 make run
+
+# TIMEOUT=<seconds>: Set the timeout for the run script. You may want to override the default value if your development environment is a low-end machine (ie: old CPU, HD disk, low RAM).
```

Comment:
@LucasGdosR could you move this description to the section "List of Optional Run Parameters"?

Let's keep the section with default parameters as simple as possible, so people who are looking for copy-and-paste are happy :)

Reply:
Moved to the correct section and split the copy-paste code into two so people can copy just the bit they need.

Reply:
Looks good, thank you!